presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -127,15 +129,15 @@ public static long resolveSnapshotId(Table table, long snapshotId)
                 .orElseThrow(() -> new PrestoException(ICEBERG_INVALID_SNAPSHOT_ID, format("Invalid snapshot [%s] for table: %s", snapshotId, table)));
     }
 
-    public static Optional<Long> resolveSnapshotIdByName(Table table, IcebergTableName name)
+    public static Optional<Long> resolveSnapshotIdByName(Table table, IcebergTableName name, Logger log)
```

Comment:
Just use a local logger in `IcebergUtil`

Reply:
🤦‍♀️ of course. Done!